frontend/src/components.css

```
@@ -0,0 +1,2546 @@
+
+@keyframes moex-pulse {
+  0%, 100% { transform: scale(1); box-shadow: 0 0 20px rgba(107, 92, 246, 0.4), 0 0 60px rgba(107, 92, 246, 0.2); }
+  50%       { transform: scale(1.05); box-shadow: 0 0 40px rgba(107, 92, 246, 0.7), 0 0 100px rgba(107, 92, 246, 0.35); }
+}
+
+.moex-shell {
+  display: flex;
+  width: 100%;
+  min-height: 100vh;
+  min-width: 900px;
+  font-family: 'Inter', -apple-system, system-ui, sans-serif;
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)